array-manipulation-3/src/flatten.js

```diff
@@ -0,0 +1,4 @@
+/* exported flatten */
+function flatten(array) {
+  return array.reduce((acc, value) => acc.concat(value), []);
```

Comment:
Can you write this using loops and conditionals?

Alternatively, if you can explain to me how this works in your own words I'll allow it

Reply:
The function flatten takes in an array as it's argument and the accumulator is initialized as an empty array. The .reduce method is then applied to all elements which then returns a single value that is the accumulated result. The .concat method then combines elements from the accumulated/current array.